frb_codegen/src/generator/dart/ty_enum.rs

```diff
@@ -152,8 +164,16 @@ impl TypeDartGeneratorTrait for TypeEnumRefGenerator<'_> {
                             format!("{{ {} }}", fields.join(""))
                         }
                     };
+                    let implements_exception = {
+                        if self.ir.is_exception && has_backtrace {
+                            "@Implements<FrbBacktracedException>()"
+                        } else {
+                            ""
+                        }
+                    };
```

Comment:
uber-nit: maybe the outermost `{}` not needed

Reply:
I didn't understand

Reply:
Not sure whether this grammar is sufficient:

```suggestion
                    let implements_exception = if self.ir.is_exception && has_backtrace {
                            "@Implements<FrbBacktracedException>()"
                        } else {
                            ""
                        };
```